GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/Controllers/Account/AccountCreationController/AccountCreationTests.cs

```diff
@@ -42,24 +42,22 @@ public void POST_Verification_Email_Is_Sent_After_Creating_User_Account()
             // Required to mock out the Url object when creating the verification URL
             controller.AddMockUriHelperNew(new Uri("https://localhost:44371/mockURL").ToString());
 
-            var mockNotifyEmailQueue = new Mock<IQueue>();
-            Program.MvcApplication.SendNotifyEmailQueue = mockNotifyEmailQueue.Object;
-            mockNotifyEmailQueue
-                .Setup(q => q.AddMessageAsync(It.IsAny<NotifyEmail>()));
+            NewUiTestHelper.MockBackgroundJobsApi
```

Comment:
Just when I thought this PR couldn't get any better. It uses my `NewUiTestHelper` 😍